### ISSUE TITLE
http://localhost service worker error

### ISSUE THREAD

**Issue by DanielAsher** [finos/perspective#1179]
## Support Question

I am using express to serve a simple page with perspective from CDN using [these](https://perspective.finos.org/docs/md/installation.html#from-cdn) instructions. When browsing to (http://localhost:8080) using Chrome 84 perspective is requesting a resource at (blob:http://localhost:8080/14eca5da-6b84-433a-83a3-5621e7b73439) that is unavailable. The DevTools Network panel shows:
<img width="1147" alt="http-service-worker-error" src="https://user-images.githubusercontent.com/7915553/92332259-79e2ca00-f074-11ea-8b29-e72727bcdab8.png">

The DevTools Source panel shows:

<img width="1151" alt="Screenshot 2020-09-06 at 19 11 01" src="https://user-images.githubusercontent.com/7915553/92332341-33419f80-f075-11ea-95af-2312ef01013b.png">

Is this a limitation of using http and service workers or a simple misunderstanding of mine?

Thanks for an awesome library!

**Comment by DanielAsher** [comment]
This seems related to https://github.com/finos/perspective/issues/1139. 

It was fixed by replacing (pug format):
```pug
  script(src='https://unpkg.com/@finos/perspective')
  script(src='https://unpkg.com/@finos/perspective-viewer')
  script(src='https://unpkg.com/@finos/perspective-viewer-datagrid')
  script(src='https://unpkg.com/@finos/perspective-viewer-d3fc')
```
with
```pug
  script(src='https://unpkg.com/@finos/perspective/dist/umd/perspective.js')
  script(src='https://unpkg.com/@finos/perspective-viewer/dist/umd/perspective-viewer.js')
  script(src='https://unpkg.com/@finos/perspective-viewer-datagrid')
  script(src='https://unpkg.com/@finos/perspective-viewer-d3fc')
```

**Comment by texodus** [comment]
This unfortunate regression has also been fixed, to be released in v0.5.3
